src/provider/systemdmanager/manager.rs

```diff
@@ -218,8 +233,14 @@ impl SystemdManager {
             return Err(disable_error);
         }
 
-        debug!("Removing unit [{}] from systemd", unit);
-        self.delete_unit_file(&unit)?;
+        // If we are not linking to the unit file but writting it directly in the
+        // units folder it won't be removed by the call to unlinkunitfiles, so we
```

Comment:
unlinkunitfiles doesn't exist, which method are you referring to? delete_unit_file?

```suggestion
        // units folder it won't be removed by the call to `delete_unit_file`, so we
```

Reply:
I was referring to the dbus method here, but misspelled that as well :)